customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDb.java

```diff
@@ -42,6 +44,7 @@ private CustomerDb(Builder builder) {
         setInstitutionDns(builder.institutionDns);
         setFeideOrganizationId(builder.feideOrganizationId);
         setCristinId(builder.cristinId);
+        setVocabularySettings(builder.vocabularySettings);
```

Comment:
Two thoughts here:

  - can there be >1 defaults? Verify with PO.
  - are there other requirements on the vocab collection? (e.g. Set, ergo no duplicates — albeit unspecced? and untested; or max entries? Default initialisation? Probably more things I can't imagine off-hand)

Reply:
Good input.

Reply:
More than 1 default allowed, confirmed by @garshol